### PR TITLE
Add open folder in addon context menu

### PR DESCRIPTION
**Pull request by Xathz**
Never worked with WPF before so not 100% sure I did the xaml correctly but tested and it works fine.

Note: I had to add `System.ComponentModel.Annotations` for it to build

### CODE REVIEW

File: WowUp.Common/WowUp.Common.csproj [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
     <PackageReference Include="protobuf-net" Version="3.0.29" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
```
Comment (jliddev) [inline WowUp.Common/WowUp.Common.csproj:10]: Can we remove this? I was able to build the PR without it

File: WowUp.WPF/Extensions/AddonExtensions.cs [hunk 2]
```diff
@@ -101,5 +102,7 @@ public static AddonDisplayState GetDisplayState(this Addon addon)
 
             return AddonDisplayState.Unknown;
         }
+
+        public static string GetFullFolderPath(this Addon addon, IWarcraftService warcraftService) => Path.GetFullPath(Path.Combine(warcraftService.GetAddonFolderPath(addon.ClientType), addon.FolderName));
```
Comment (jliddev) [inline WowUp.WPF/Extensions/AddonExtensions.cs:106]: This should probably go in the IWarcraftService

File: WowUp.WPF/Utilities/FileUtilities.cs [hunk 3]
```diff
@@ -168,5 +168,12 @@ public static void DirectoryCopy(string sourceDirName, string destDirName, bool
                 }
             }
         }
+
+        public static void OpenInFileManager(string path)
```
Comment (jliddev) [inline WowUp.WPF/Utilities/FileUtilities.cs:172]: This function seems fine, but can we use the same format as StringExtensions.OpenUrlInBrowser which does basically the same thing.

File: WowUp.WPF/Views/AddonsView.xaml [hunk 4]
```diff
@@ -267,6 +267,9 @@
                                       Command="{Binding AlphaCheckedCommand}"></MenuItem>
                         </MenuItem>
                         <Separator Visibility="{Binding ShowUpdateButton, Converter={StaticResource BoolToVisibilty}}" />
+                        <MenuItem Header="Open Folder"
```
Comment (jliddev) [inline WowUp.WPF/Views/AddonsView.xaml:270]: This is fine, maybe move it above the Separator tag so its in the same place regardless of update status.

**Review by jliddev — CHANGES_REQUESTED** [review]
Feature works perfectly, just some minor changes before merge. Also pull in latest Develop branch changes if you can so there's no merge conflicts.

**Comment by Xathz** [comment]
My bad didn't even see a dev branch, will close and resubmit the pr on the correct branch.

**Comment by jliddev** [comment]
Nice work :D

**Pull request closed without merging**